src/runtime_src/tools/xclbinutil/XclBinUtilities.h

```diff
@@ -37,6 +37,17 @@ typedef enum {
   XET_MISSING_SECTION = 100, // Section is missing
 } XclBinExceptionType;
 
+//---------------------------------------------------------------------------
+// TODO: Make the TODO more friendly:
+//       1. Add an environment variable to enable / disable it
+//       2. Remove the extra "note:" output
+//       3. Refactor the code to be like a method instead (e.g., TODO("Do something"))
+#define Stringize( L )     #L 
+#define MakeString( M, L ) M(L)
+#define $Line MakeString( Stringize, __LINE__ )
+#define TODO __FILE__ "(" $Line ") : TODO: "
```

Comment:
Names of these macros seem too generic in a header file ?

Reply:
Thank you for catching this.   I used this macro in a pragma message to help remind me of work I still need to do in the codebase before I create a pull request.   For example:

```
#pragma message(TODO "Simple build todo reminder.")
Produces the following output:

/proj/xcohdstaff1/stephenr/github/XRT/master/src/runtime_src/tools/xclbinutil/XclBinUtilMain.cxx: In function 'int main_(int, const char**)':
/proj/xcohdstaff1/stephenr/github/XRT/master/src/runtime_src/tools/xclbinutil/XclBinUtilMain.cxx:607:51: note: #pragma message: /proj/xcohdstaff1/stephenr/github/XRT/master/src/runtime_src/tools/xclbinutil/XclBinUtilMain.cxx(607) : TODO: Simple build todo reminder.
 #pragma message(TODO "Simple build todo reminder.")

```
Ideally, it should only produced:
`/proj/xcohdstaff1/stephenr/github/XRT/master/src/runtime_src/tools/xclbinutil/XclBinUtilMain.cxx(607) : TODO: Simple build todo reminder.
`But because we have the gcc in a high verbosity mode, we get some gcc debug messages as well.

This code should not be been submitted.  That said, if you have another idea on how I might inserted compiler INFO messages, please do let me know :).